webpack.config.js

```diff
@@ -30,20 +32,15 @@ module.exports = {
           plugins: ['transform-runtime']
         }
       },
-      { test: /\.css$/,
-        include: path.resolve(__dirname, 'static_src/css'),
+      {
+        test: /\.css$/,
+        include: [
+          path.resolve(__dirname, 'static_src/css'),
+          path.resolve(__dirname, 'node_modules/cloudgov-style'),
+          CG_STYLE_PATH
```

Comment:
Yes, everything will break. That's because the JSX components require the styles and if they can't resolve those file names the webpack process will fail